src/test/java/org/opentripplanner/middleware/ApiKeyManagementTest.java

```diff
@@ -61,13 +61,13 @@ public static void setUp() throws IOException, InterruptedException {
      */
     @AfterAll
     public static void tearDown() {
-        assumeTrue(isEndToEnd);
```

Comment:
I think tearDown should `assumeTrue(isEndToEnd)`, right? That's how our other test classes are constructed I believe. I think that would allow you to remove the conditional checks for null apiUser and adminUser.

Reply:
@landonreed I think so. I have updated, it runs fine locally so will keep an eye on CI.